### PR TITLE
Boilerplate subscribe  bb13099762b9a98c72498c2d6e3d7a3672e73be2

### PR DESCRIPTION
**Pull request by ritmun**
Updated to new convention in boilerplate to handle the build and push of images to the registry instead of the local scripts in 'hack' folder.

Convention is added to osd-network-verifier
After migration, unused build/deploy scripts are removed from the repository
Do not merge until these dependencies are merged
app-interface PR dependency https://gitlab.cee.redhat.com/service/app-interface/-/merge_requests/38470/diffs
Release PR dependency https://github.com/openshift/release/pull/28359

Continued from PR https://github.com/openshift/osd-network-verifier/pull/76

### CODE REVIEW

**Review by bdematte — COMMENTED** [review]
Testing the PR, looks like the statement isn't effective here so should be purely removed and follow-up with boilerplate maintaining team on rather we should have the default enforced by a convention

**Comment by bdematte** [comment]
/retest
Fixed the formating issues

File: Makefile [hunk 1]
```diff
@@ -1,14 +1,15 @@
...
-include boilerplate/generated-includes.mk
-
-SHELL := /usr/bin/env bash
+include project.mk
 
 # Include shared Makefiles
-include project.mk
-include standard.mk
+include boilerplate/generated-includes.mk
+default: build
```
[the remark this answers is not in the document]
Comment (ritmun) [reply Makefile]: removed

File: README.md [hunk 2]
```diff
@@ -21,11 +21,16 @@ It currently verifies:
 
 
 ## Makefile Targets
-- `make build`: Builds executable
```
Comment (bdematte) [inline README.md:24]: `make build` is still a valid target, so shouldn't remove it

File: README.md [hunk 3]
```diff
@@ -21,11 +21,16 @@ It currently verifies:
...
-- `make build-push`: Builds and pushes image from build/ to ` quay.io/app-sre/osd-network-verifier:$(IMAGE_URI_VERSION)`
-- `make skopeo-push`: (TODO add)
-- 
+ONV uses openshift/boilerplate https://github.com/openshift/boilerplate
+
+Some useful targets
+- `make osd-container-image-build` : Build the default container at build/Dockerfile and tag it based on the commit. Specify DOCKERFILE and IMAGE_URI to build other containers.
+- `make osd-container-image-push` : Push the default container. 
+  - To run locally, `make REGISTRY_USER=<quay-username> REGISTRY_TOKEN=<quay-token>  osd-container-image-build-push`
+- `make prow-config` : Updates the corresponding Prow config file in openshift/release to run make test on merge requests. This test make target should be defined by the consumer. If this is a new repository it should be onboarded to openshift/release first before this is run.
```
Comment (bdematte) [inline README.md:30]: I am not sure we should document this command as it is only an onboarding helper generating a part of the global prow config (specific to 1 convention). 
If we want to keep it documented, we should be clearer on what it does and what it doesn't do.

File: README.md (hunk 2 above)
Comment (ritmun) [reply README.md:24]: thanks, good catch..

**Review by fahlmant — APPROVED** [review]
/approve

**Pull request merged**